### DISCUSSION TITLE
🚨 README before opening a discussion 🚨

### DISCUSSION THREAD

**Discussion by mr-manuel** [mr-manuel/venus-os_dbus-serialbattery#119] [Announcements]
Hello and welcome to our community! 👋

To help us assist you better, please follow these guidelines:

* **Reporting a Bug:** If you've found a bug, please [open a bug report](https://github.com/mr-manuel/venus-os_dbus-serialbattery/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml).
* **Support Request:** If you need help, please [open a support request](https://github.com/mr-manuel/venus-os_dbus-serialbattery/issues/new?assignees=&labels=support&projects=&template=support_request.yml).
* **Feature Request:** If you have an idea for a new feature, please [open a feature request](https://github.com/mr-manuel/venus-os_dbus-serialbattery/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yml).

For anything else, feel free to start a discussion. Please note that the maintainer may not be able to respond to all discussions.

**Comment by kemills1** [comment]
Hi,
I’ve just ventured into BMS control of my Victron system. Looking at the SerialBattery features I see it can limit the discharge current. I understand how the BMS can limit charge current by throttling the charge controllers but how can it limit the discharge current that the inverter takes?

**Reply by mr-manuel** [reply]
That only works, if you have a secondary power source other than the battery and you are using a Victron inverter. Then the inverter is limited and the rest is taken from the grid/generator.

**Reply by kemills1** [reply]
OK, so if I use DCCM_CV_ENABLE = False, with this stop discharge throttling? The system is off grid so there is no alternative power available.

**Reply by mr-manuel** [reply]
If you are off grid, then this value only takes action, if it is 0. Then the system turns the inverter off.

**Comment by kemills1** [comment]
I have a couple of questions. I’ve connected a JKBMS to my Raspberry Pi using a 485 to USB adapter. It mostly seems to work but charging is being prevented because the battery temp, which is measured at 6°C, is too low. What is the threshold set to and how can I change it? The JKBMS cutoff is set to 2°C.
